docs/3.0/common/reloadable.md

```diff
@@ -0,0 +1,17 @@
+## Reloadable
```

Comment:
Instead of adding it as a "chapter" to the page, it would be more suitable to add it as an option like `link_to_child_resource`.
Check `./../common/associations_link_to_child_resource_common.md`.

That way it will be highlighted as such in the sidebar.
Please see how that is implemented.

Also please see what structure do the options follow (default value, possible values, etc.).

Reply:
Ok, thanks.